### ISSUE TITLE
Downloading hparams table should include the Trial ID column

### ISSUE THREAD

**Issue by jcolen** [tensorflow/tensorboard#5498]
Downloading the hparams table as a csv omits the Trial ID column. It would be helpful if this was included along with the logged hyperparameters and metrics for completeness.

**Comment by bileschi** [comment]
Thanks for the suggestion!   Just for clarity, can you explain what you would use the trialID for?  Is it to reference back into the TensorBoard UI?  Is it for further analysis?

**Comment by sidphbot** [comment]
Agree with OP, tensorboard is mighty helpful but sometimes you have a bunch of experiments where hp_metric was not set properly or simply we want a detailed further analysis.
The csv helps but as you said, we do need a way to find actual names/paths to reuse checkpoints or models for further analysis or even re-training on downstream tasks (after downloaded csv analysis, I mean)

**Comment by EClemMarq** [comment]
Like @sidphbot, I would use this to find model paths and checkpoints.  For example, when I do a hyper-parameter tuning run, I have to go to TensorBoard every time I want to find the path for my highest performing trial.  The downloaded CSV will tell me what my highest accuracy was and which hyper-parameters I used to get it, but not where to find the saved weights.

I would also use this to evaluate how well my hyper-parameter tuning run is going.  If I'm using a Bayesian method, are the hyper-parameters for trial 100 better than the hyper-parameters for trial 2, etc.?  Without the trial ID in the .csv, I can't do that.

**Comment by attilamester** [comment]
any updates on this thread? Including trial ID at least with optional checkbox, would be really helpful.
